layouts/partials/tabs.html

```diff
@@ -60,9 +60,12 @@ <h3 class="subtitle is-5 is-muted">{{ $subtitle }}</h3>
 
         <ul class="cd-tabs__panels">
         <li id="tab-inbox" class="cd-tabs__panel cd-tabs__panel--selected text-component">
-            <p>Visualization Lorem ipsum dolor sit amet, consectetur adipisicing elit. Earum recusandae rem animi accusamus quisquam reprehenderit sed voluptates, numquam, quibusdam velit dolores repellendus tempora corrupti accusantium obcaecati voluptate totam eveniet laboriosam?</p>
-
-            <p>Inbox Lorem ipsum dolor sit amet, consectetur adipisicing elit. Earum recusandae rem animi accusamus quisquam reprehenderit sed voluptates, numquam, quibusdam velit dolores repellendus tempora corrupti accusantium obcaecati voluptate totam eveniet laboriosam?</p>
+                <img src="images/content_images/visualization.gif" alt="visualization figure" align="left" height="300" width="300" style="margin: 10px 25px; border-radius: 6px"> 
+                <p>Data Visualization refers to graphical representation of textual data that makes it human-readable, easier to understand and interpret by exposing patterns, trends and correlations within the data. Data visualization paves the way towards absorbing data and gaining actionable insights. It enables advanced data analytics for identifying areas that need attention and improvement. Users can make data-driven decisions and analyse huge volumes of data through visual elements such as graphs, bar, pie and line charts, maps, infographics, dashboards, geographic maps, heatmaps etc.</p>
+                <p>Data Scientists rely upon data visualization to manipulate, analyse and query large volumes of data using interactive images, visual analytics and gain useful insights into the data that cannot be otherwise detected in textual form.</p>
```

Comment:
I think starting with a general sentence which is "what is data visualization" (as you did) is good. But after that I think it would be good to switch to what that looks like in Python. Along the lines of:
- Matplotlib is the most-used Python visualization package. It is a great for making 2-D static plots. 
- Seaborn builds on top of Matplotlib for statistical plots
- Bokeh, Plotly and Altair are the most-used packages for dynamic, browser-based plotting.
- The Holoviz suite of packages provides a higher-level plotting API 
- For 3-D viz there's VisPy, Napari and Glue

Reply:
I think the https://julialang.org/ tabs content is quite nice - it manages to give a snapshot of both what a topic is about, and the main packages one can use for it.

Reply:
Sure. done.

Reply:
The [pyviz.org website](https://pyviz.org/overviews/index.html) gives a nice,  overview of visualization. It is a bit of a work-in-progress, but might give some inspiration

Reply:
Thanks @mattip.  Inspired by pyviz - here is what I came up with:  Ref: https://github.com/numpy/numpy.org/pull/64/commits/aad18a0f52a7b862570a9aad4f1d11987dad3647 
<img width="985" alt="aad18a0" src="https://user-images.githubusercontent.com/5890484/68127523-12338e00-ff3c-11e9-9043-9d1029f41aa5.png">

Reply:
I like those visuals! This is pretty close. We should probably tone down the performance claims a little, it's not all NumPy-based (e.g. OpenGL in Vispy/Napari) and the C/Fortran is an implementation detail that's probably best left out here.

Reply:
Ok, will take care of that. Thx for the inputs!